versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 0.12"
+  required_version = ">= 0.12.6"
```

Comment:
I really don't like forcing people to a minimum patch version. Is there something in 0.12.6 that isn't available in 5 or lower that we absolutely need?

Reply:
[`for_each`](https://www.terraform.io/docs/configuration/resources.html#for_each-multiple-resource-instances-defined-by-a-map-or-set-of-strings)

> Version note: `for_each` was added in Terraform 0.12.6.

Reply:
Got it makes sense.